libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -457,6 +733,10 @@ void AP_UAVCAN::rc_out_sem_give()
 
 void AP_UAVCAN::do_cyclic(void)
 {
+    static uint64_t fix_out_last_send_time = AP_HAL::millis64();
```

Comment:
these static's are not allowed. Please move the variables to the class definition